superduperdb/ext/transformers/training.py

```diff
@@ -242,6 +236,12 @@ class LLMTrainer(TrainingArguments, SuperDuperTrainer):
     ray_configs: t.Optional[dict] = None
 
     def __post_init__(self, db, artifacts):
+        # TODO: Only need to check the required packages for initializing the class
+        requires_packages(
```

Comment:
We need this to build a requirements file `python -m superduperdb requirements`

Reply:
OK, I will move it back.

Reply:
Done